.github/workflows/publish-docker-image.yml

```diff
@@ -17,10 +17,18 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Get current Studio version
+        id: get-studio-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
+        with:
+          path: apps/studio
+
       - name: Build and push Docker image
         uses: docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4
         with:
           context: .
           file: ./apps/studio/Dockerfile
           push: true
-          tags: asyncapi/studio:latest
\ No newline at end of file
+          tags: |
+            asyncapi/studio:latest
+            asyncapi/studio:${{ steps.get-studio-version.outputs.current-version}}
```

Comment:
Syntax as mentioned in https://github.com/docker/build-push-action?tab=readme-ov-file#inputs